Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -1117,8 +1124,23 @@ int function ServerSortLogic ( serverStruct a, serverStruct b )
 	bool direction
 
 	// We can hard code this cause adding entire columns isn't as easy
-	switch ( filterDirection.sortingBy )
+	switch ( filterDirection.sortingBy ) 
 	{
+		case sortingBy.DEFAULT:
+			aTemp = a.serverPlayers
+			bTemp = b.serverPlayers
+			if (aTemp >= a.serverPlayersMax -1)
+				if (aTemp == a.serverPlayersMax)
+					aTemp = -2
+				else
+					aTemp = -1
+			if (bTemp >= b.serverPlayersMax - 1)
+				if (bTemp == b.serverPlayersMax)
+					bTemp = -2
+				else
+					bTemp = -1
+			direction = filterDirection.serverName
+			break;
```

Comment:
I'd do this then ^^

```suggestion
		case sortingBy.DEFAULT:
			aTemp = a.serverPlayers
			bTemp = b.serverPlayers

			// `1000` is assumed to always be higher than `serverPlayersMax`
			if (aTemp + 1 < a.serverPlayersMax)
				aTemp = aTemp+1000
			if (bTemp + 1 < b.serverPlayersMax)
				bTemp = bTemp+1000

			direction = filterDirection.serverName
			break;
```

Reply:
![mp_lobby0009](https://user-images.githubusercontent.com/40122905/164797584-ec999ac7-ee84-4246-b0be-b032ac300d72.jpg)

Reply:
Except that that doesnt work. It mixes the almost full servers with full servers. There's a reason i wrote that `if`...

Reply:
How about:

```suggestion
		case sortingBy.DEFAULT:
			aTemp = a.serverPlayers
			bTemp = b.serverPlayers

			// `1000` is assumed to always be higher than `serverPlayersMax`
			if (aTemp + 1 < a.serverPlayersMax)
				aTemp = aTemp+2000
			if (bTemp + 1 < b.serverPlayersMax)
				bTemp = bTemp+2000
			if (aTemp + 1 == a.serverPlayersMax)
				aTemp = aTemp+1000
			if (bTemp + 1 == b.serverPlayersMax)
				bTemp = bTemp+1000

			direction = filterDirection.serverName
			break;
```

Reply:
That way "full" and "nearly full" are still sorted internally ^^

![mp_lobby0010](https://user-images.githubusercontent.com/40122905/164799192-5dd950d3-0144-4c43-9111-11db8f98c089.jpg)